charts/rhdh/templates/model-catalog-rbac.yaml

```diff
@@ -0,0 +1,95 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rhdh-rhoai-bridge
+  namespace: rolling-demo-ns
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: rhdh-rhoai-bridge
+  annotations:
+    argocd.argoproj.io/sync-wave: "0"
+rules:
+  - apiGroups:
+      - apiextensions.k8s.io
+    resources:
+      - customresourcedefinitions
+    verbs:
+      - get
+  - apiGroups:
+      - route.openshift.io
+    resources:
+      - routes
+    verbs:
+      - get
+      - list
+      - watch
+
+  - apiGroups: ["serving.kserve.io"]
+    resources: ["inferenceservices"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["apps"]
+    resources: ["deployments"]
+    verbs: ["get", "list", "watch", "patch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: rhdh-rhoai-bridge
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rhdh-rhoai-bridge
+subjects:
+  - kind: ServiceAccount
+    name: default
+    namespace: rolling-demo-ns
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: rhdh-odh-dashboard-permissions
+  namespace: odh-model-registries
```

Comment:
when we pivot to using RHOAI this will change  .... see https://github.com/redhat-ai-dev/odh-kubeflow-model-registry-setup/tree/main/kustomize-rhoai and compare with https://github.com/redhat-ai-dev/odh-kubeflow-model-registry-setup/tree/main/kustomize 

you may want to parameterize this ... or just go with the rhoai version as the dev cluster will not be going to odh anytime soon

@johnmcollier FYI

Reply:
I have adopted in [623d936](https://github.com/redhat-ai-dev/ai-rolling-demo-gitops/pull/5/commits/623d936078a562d5cd16c7330dbcff8a2f9c1842) & [10c8f33](https://github.com/redhat-ai-dev/ai-rolling-demo-gitops/pull/5/commits/10c8f33fcf3a194671ff2bb21b7a08c94b91fca4) the configurations introduced by https://github.com/redhat-ai-dev/model-catalog-bridge/pull/26

In my tests looks good. I'm able to register a test model in RHOAI, the sidecar containers automatically import and update the locations in model-catalog.json and finally I can see the model in RHDH.